### ISSUE TITLE
BUG: Number sliders on pdf import not updating

### ISSUE THREAD

**Issue by Luk3Pl4ys** [zsviczian/obsidian-excalidraw-plugin#1716]
**Your environment**
SYSTEM INFO:
    Operating system: android 14 (samsung SM-P613)
    Obsidian version: 1.5.12 (134)
    API version: v1.5.12
    Login status: not logged in
    Live preview: on
    Base theme: dark
    Community theme: none
    Snippets enabled: 1
    Restricted mode: off
    Plugins installed: 2
    Plugins enabled: 2
        1: Remotely Save v0.4.16
        2: Excalidraw v2.1.4

**Describe the bug**
The sliders when importing a pdf file to the canvas no longer update their values while dragging them, making it hard to select a specific value. Once you stop draging the value updates as usual.

**To Reproduce**
1. Click on "Insert ANY file"
3. Select any pdf file
4. Click on "as Pdf"
5. Drag any slider: "Number of columns", "Size of gap between pages", "Imported page size"
6. The value doesn't update while dragging but only when dragging stops

**Expected behavior**
The value of the slider should update while still draging to allow for easy and precise selecting of the value you want. This was already the case in earlier version and was working till 2.1.2 as far as i can remember.

**Comment by zsviczian** [comment]
Yes, I noticed as well. Thanks for raising an issue - this helps me not to forget to fix this.
